docs-2.0/3.ngql-guide/16.subgraph-and-path/2.find-path.md

```diff
@@ -0,0 +1,53 @@
+# FIND PATH
+
+```ngql
+FIND SHORTEST | ALL | NOLOOP PATH FROM <vertex_id_list> TO <vertex_id_list>
+OVER <edge_type_list> [REVERSELY] [BIDIRECT] [UPTO <N> STEPS]
+
+<vertex_id_list> ::=
+    [vertex_id [, vertex_id ...]
+```
+
+The `FIND PATH` statement finds the paths between the selected source vertex and destination vertex.
+
+- `SHORTEST` finds the shortest path.
+- `ALL` finds all the paths.
+- `<vertex_id_list>` is a list of vertex IDs separated by commas (,). It supports `$-` and `$var`.
+- `<edge_type_list>`  is a list of edge types separated by commas (,). `*` is all edge types.
+- `<N>` is the hop number. The default value is 5.
+
+## Limitations
+
+- When source and destination vertices are ID lists, the shortest path between any source vertices and the destination vertices is returned.
```

Comment:
"When source and destination vertices are ID lists" ==> When a list of source and/or destination vertex IDs are specified"?

Reply:
nice!